libs/deso-protocol/src/lib/user/User.ts

```diff
@@ -29,7 +33,17 @@ export class User {
     this.identity = identity;
   }
 
-  public async getUserStateless(
+  public async getUsernameForPublicKey(
+    PublicKeyBase58Check: string
+  ): Promise<any> {
+    return (
+      await axios.get(
+        `${this.node.getUri()}/get-user-name-for-public-key/${PublicKeyBase58Check}`
+      )
+    ).data;
+  }
+
+  public async getUsersStateless(
```

Comment:
for backwards compatibility, we should still have a `getUserStateless` function that basically just passes through to `getUsersStateless`. We can remove it when we have a new major version release.